AzureIoT_StoreAndForward/README.md

```diff
@@ -0,0 +1,118 @@
+| `copy_required_files.cmd` | Windows script to copy files from the SimpleFileSystem_RemoteDisk project |
+| `copy_required_files.sh` | Linux script to copy files from the SimpleFileSystem_RemoteDisk project |
+| `README.md` | This README file. |
+| `LICENSE.txt`   | The license for the project. |
+
+## Prerequisites & Setup
+
+- An Azure Sphere-based device with development features (see [Get started with Azure Sphere](https://azure.microsoft.com/en-us/services/azure-sphere/get-started/) for more information).
+- Setup a development environment for Azure Sphere (see [Quickstarts to set up your Azure Sphere device](https://docs.microsoft.com/en-us/azure-sphere/install/overview) for more information).
+
+Note that the Azure Sphere High Level application is configured for the 21.07 SDK release.
+
+## How to use
+
+You will need the following:
+* Azure Sphere device claimed into a tenant
+* A copy of the [Azure Sphere Github samples repository](https://github.com/Azure/azure-sphere-samples) (specifically the [Azure IoT sample](https://github.com/Azure/azure-sphere-samples/tree/master/Samples/AzureIoT))
+* A copy of the [Azure Sphere Github Gallery repository](https://github.com/Azure/azure-sphere-gallery)
+
+Copy the files from this project `src` to your copy of the root of the Azure Sphere `AzureIoT` sample project.
+
+You will need to copy files from the Azure Sphere Gallery `SimpleFileSystem_RemoteDisk` project:
+* On **Windows**, open a command prompt to this project folder and run `copy_required_files.cmd`
+* On **Linux**, open a terminal and run `copy_required_files.sh` 
+
+The store and forward gallery project modifies several files:
+
+`AzureIoT/CMakeLists.txt` - You will need modify the PC_HOST_IP definition within CMakeLists.txt to include the IP address of your PC (the PC there the Python remote disk host will run).
+
+```cmd
+add_compile_definitions(PC_HOST_IP="YOUR_PC_IP_ADDRESS")
+```
+
+`AzureIoT/app_manifest.json` - add the IP address of your PC to the `AllowedConnections` list in `app_manifest.json`
+
+`AzureIoT/common/exitcodes.h` - a new exit code `Exit_Code_Init_FileSystem` has been added.
+
+`AzureIoT/common/main.c` - The AzureIoT sample uses the `telemetryUploadEnabled` boolean to define whether telemetry will be uploaded or not, the sample has been extended to store telemetry in the simple file system when telemetryUploadEnabled == false. 
+
+The original AzureIoT sample uses a 5 second tick to generate telemetry, the tick has been modified to be 1 second. Every five seconds new telemetry will be generated - if there are no stored telemetry items and telemetryUploadEnabled is true the new telemetry will be uploaded, if there are existing stored telemetry items the new telemetry will be written to the simple file system (this preserves date/time order of uploaded telemetry). For the other four seconds stored telemetry will be uploaded if telemetryUploadEnabled is true. 
```

Comment:
I find this paragraph a little confusing.  Right after you say "the tick has been modified to be 1 second", you say "Every five seconds...".   If you read the whole thing it kind of makes sense, but what about changing that second sentence to something like "Every fifth tick new telemetry will be generated ...".  Minor point, but I think that would be clearer.

Reply:
agreed, readme.md is updated.